CHANGELOG.md

```diff
@@ -7,8 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+
```

Comment:
Good idea to update the changelog :+1: 

It should go here (in Unreleased) rather than in 1.1.0 (already released):

```suggestion
### Changed

- Redesign the wheel hub (thanks to @Tordjx and @EtienneAr)
```